jwt/__init__.py

```diff
@@ -359,6 +359,17 @@ def load(jwt):
 
 def verify_signature(payload, signing_input, header, signature, key='',
                      verify_expiration=True, leeway=0, **kwargs):
+
+    if isinstance(leeway, datetime.timedelta):
+        try:
+            leeway.total_seconds
```

Comment:
Not sure about this one though, I usually prefer to handle things like this via exceptions.